metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/interfaces/MessageParser.java

```diff
@@ -34,15 +34,15 @@
    * @param rawMessage
    * @return If null is returned, this is treated as an empty list.
    */
-  List<T> parse(byte[] rawMessage);
+  List<T> parse(byte[] rawMessage, SensorParserConfig sensorParserConfig);
```

Comment:
Are there any user-developed parsers out there that don't subclass either BasicParser or GrokParser?  If so, they'll break on the new call to the new method in MessageParser.  If this is a concern, it can be avoided by making "ConfiguredMessageParser extends MessageParser" and putting the new configurationUpdate() method there.  GrokParser would be the only current implementation of ConfiguredMessageParser, and the call to parser.configurationUpdate() in ParserBolt would be made conditional on "(parser instanceof ConfiguredMessageParser)"